gallery/plot_video_api.py

```diff
@@ -325,7 +325,7 @@ def __iter__(self):
 # ----------------------------------
 # Example of visualized video
 
-import matplotlib.pylab as plt
+import matplotlib.pyplot as plt
```

Comment:
pylab is deprecated : https://matplotlib.org/stable/api/index.html?highlight=pylab#module-pylab